modules/P20_firmware_bin_extractor.sh

```diff
@@ -308,11 +308,11 @@ binwalking() {
     if [[ $IN_DOCKER -eq 1 ]] ; then
       cd / || return
       print_output "$(binwalk -E -F -J "$FIRMWARE_PATH_BAK")"
-      mv "$(basename "$FIRMWARE_PATH".png)" "$LOG_DIR"/"$(basename "$FIRMWARE_PATH"_entropy.png)" 2> /dev/null
+      mv "$(basename "$FIRMWARE_PATH".png)" "$LOG_DIR"/"$(basename firmware_entropy.png)" 2> /dev/null
```

Comment:
Change it to following line (basename isn't needed in your case):
```
mv "$(basename "$FIRMWARE_PATH".png)" "$LOG_DIR""/firmware_entropy.png" 2> /dev/null
```
Same in line 315.